cves/kernel/CVE-2020-9391.yml

```diff
@@ -221,8 +225,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: lib
+  note: This issue is related to the library function brk() as well as having direct
+    effects on malloc.
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
```

Comment:
I think the professor wants some sort of note describe why there may be no really interesting commits (maybe due to the trivial nature of the issue)

Reply:
Same should go for the other yml if it does not have one